src/Integration/Binding/CSharpVBProjectBinder.cs

```diff
@@ -156,10 +157,27 @@ private RuleSet FindDeclarationRuleSet(Project project, RuleSetDeclaration decla
             return ruleSetSerializer.LoadRuleSet(ruleSetFilePath);
         }
 
-        private bool ProjectHasAdditionalFile(Project project, string additionalFilePath)
+        public bool DoesProjectReferenceAdditionalFile(Project project, string solutionAdditionalFilePath)
         {
-            // ritag todo: 1. Check if there is another SonarLint.xml, 2. Check if the property is AdditionalFiles
-            return projectSystemHelper.IsFileInProject(project, additionalFilePath);
+            var additionalFile = projectSystemHelper.GetFileInProject(project, solutionAdditionalFilePath);
```

Comment:
I don't know if GetFileInProject behaves differently that IsFileInProject.
IsFileInProject uses underlying API `IsDocumentInProject`, while GetFileInProject simply iterates the ProjectItems.